Formula/profanity.rb

```diff
@@ -19,6 +19,8 @@ class Profanity < Formula
     depends_on "libtool" => :build
   end
 
+  option "with-python-plugin-support", "Build with support for python plugins"
```

Comment:
Sounds good, but `:python` is only required, if profanity is build with option `--with-pyton-plugin-support` - there's no need to choose python otherwise. Couldn't that be confusing.

But if you think the 'optional' solution is best, I will patch it this way.